README.md

```diff
@@ -15,7 +15,27 @@ If you don't have YunoHost, please consult [the guide](https://yunohost.org/#/in
 
 ## Overview
 
-Open source collaboration platform built for developers
+## Mattermost Team Edition
+A free-to-use, open source, self-hosted alternative to proprietary SaaS messaging. Team Edition is your open source “virtual office”, offering all the core productivity benefits of competing SaaS solutions. It deploys as a single Linux binary with MySQL under an MIT license.
+
+## Mattermost Enterprise Edition
+To explore the benefits of Mattermost’s enterprise features, you can replace the Mattermost Team Edition binary with a Mattermost Enterprise Edition binary. This version supports upgrading to Mattermost Enterprise Edition E10 or E20.
+
+Features include:
```

Comment:
It's nice to have the features list.

However with the current layout it looks like these are the features of the Enterprise edition only. How about formatting "Features" as a title maybe?